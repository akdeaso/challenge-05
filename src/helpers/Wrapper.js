import React from 'react';
import {Provider} from 'react-redux';
import {NavigationContainer} from '@react-navigation/native';
import {PersistGate} from 'redux-persist/lib/integration/react';

import {persistedStore, store} from '../store';
import {navigationRef} from './navigate';

export default function Wrapper(child) {
  return (
    <NavigationContainer ref={navigationRef}>
      <Provider store={store}>
        <PersistGate persistor={persistedStore}>{child}</PersistGate>
      </Provider>
    </NavigationContainer>
  );
}
