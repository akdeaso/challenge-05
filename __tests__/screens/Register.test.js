import * as React from 'react';
import {create} from 'react-test-renderer';
import Register from '../../src/screens/Register';
import Wrapper from '../../src/helpers/Wrapper';

const component = create(Wrapper(<Register />));

describe('should render register', () => {
  test('matching snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
