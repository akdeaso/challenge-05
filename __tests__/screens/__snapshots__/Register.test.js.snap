// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render register matching snapshot 1`] = `null`;
